src/components/slider/slider-view.tsx

```diff
@@ -179,15 +180,18 @@ export class SliderView extends React.Component<SliderViewProps, {}> {
                 aria-valuemax={`${this.props.max}`}
                 aria-valuenow={`${this.props.value}`}
                 tabIndex={this.props.disabled ? -1 : 0}
+                data-tooltip-for={idPrefix + index}
 
                 key={index}
             >
                 {this.props.displayTooltip &&
                     <Tooltip
-                        open={this.props.currentHoverIndex === index || this.props.currentHandleIndex === index}
+                        id={idPrefix + index}
+                        showTrigger="focus,mouseenter"
```

Comment:
is this an API? why?

Reply:
do you mean, why the `showTrigger` and `hideTrigger` are API props, or why does it accept the strings instead of arrays?